traveler-ios-ui/TravelerKitUI/ViewControllers/CatalogItemResultViewController.swift

```diff
@@ -131,3 +152,42 @@ extension CatalogItemResultViewController: UIScrollViewDelegate {
         }
     }
 }
+
+extension CatalogItemResultViewController: WishlistRemoveDelegate {
+    func wishlistRemoveDidSucceed() {
+        wishlistButton.isEnabled = true
+        catalogItemDetails?.isWishlisted = false
```

Comment:
all properties on models are immutable for the most part. It might be better to consider passing the wishlisted catalogItemDetails here. Maybe its not possible not entirely sure

Reply:
You didnt address this comment

Reply:
TBD